x-pack/plugins/actions/server/builtin_action_types/case/utils.ts

```diff
@@ -1,111 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { flow, get } from 'lodash';
-
-import {
-  MapRecord,
-  TransformFieldsArgs,
-  Comment,
-  EntityInformation,
-  PipedField,
-  AnyParams,
-  PrepareFieldsForTransformArgs,
-} from './types';
-
-import { transformers } from './transformers';
-
-import { SUPPORTED_SOURCE_FIELDS } from './constants';
-
-export const normalizeMapping = (supportedFields: string[], mapping: MapRecord[]): MapRecord[] => {
```

Comment:
We need to keep this functionality to prevent prototype pollution. The user is allowed to set its own mapping, right?

Reply:
not yet, its still strictly typed and not dynamic

Reply:
Ok, let's keep that in mind when we switch to the dynamic version.